src/components/UI/Card/CardHeader/CardHeaderElements.jsx

```diff
@@ -0,0 +1,54 @@
+import "./CardHeaderElements.css";
+import React, { useState } from "react";
+import { FaPen } from "react-icons/fa";
+import { FaCheck } from "react-icons/fa";
+import { FaTimes } from "react-icons/fa";
+
+const StyleChanger = (props) => {
+  const [cbState, setCbState] = useState(false); //false - dafault styles, true - dark mode
+  const [penClicked, setPenClicked] = useState(false); //true - edit mode
+
+  const changeHandler = (event) => {
+    setCbState(event.target.checked);
+    props.onCbValue(!cbState);
+  };
+
+  const penClickHandler = () => {
+    setCbState(false); //set default styles in edit mode
+    props.onCbValue(false);
+
+    setPenClicked(true); //set edit mode
+    props.onPenClicked(true);
+  };
+
+  const saveClickHandler = () => {
+    setPenClicked(false); //unset edit mode
+    props.onPenClicked(false);
+
+    props.onSaveClick();
+  };
+
+  const cancelClickHandler = () => {
+    setPenClicked(false); //unset edit mode
+    props.onPenClicked(false);
+
+    props.onCancelClick();
+  };
+
+  return (
+    <div>
```

Comment:
Also, if you are using some div / span elements just to wrap up several components and return 1 node - you can use `<React.Fragment>` hoc that do basically the same. Either way, currently becomes very popular a practice of using an empty tag as a wrapper: `<>...</>`.